swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
+from .item import Item
+
+class Clothing(Item):
+
+    def __init__(self, category='Clothing', condition=0, age=0):
+        self.category = category
+        self.condition = condition
+        self.age = age
```

Comment:
This class definition definitely works but leaves Clothing instances susceptible to having their category changed. This is where using the parent constructor is helpful to ensure that every clothing instance will have the category of 'Clothing'.

```suggestion
class Clothing(Item):

    def __init__(self, condition=0, age=0):
        super().__init__(category="Clothing", condition=condition, age=age)
```

Reply:
Additionally, we're not truly using inheritance without the parent constructor. Inheritance is helpful so that child classes don't have to recreate the attributes, they can inherit attributes from their parent but still have unique values for them.